src/actions/filesDropped/parseCsv.js

```diff
@@ -26,7 +21,7 @@ export const parseCsvTsv = async (file) => {
       },
       encoding: "UTF-8",
       comments: "#",
-      delimiter: (csv_file_types.includes(file.type)) ? "," : "\t",
+      delimiter: ",",
```

Comment:
I only glanced over the PR and it looks good overall. But this place made me wonder how is TSV handled now. Is it still working? It is unclear from code changes immediately where it migrated. Is it now handled by the `xlsx` rather than `papaparse`?

Reply:
Ah, sorry this isn't clear. Everything gets converted to a CSV string by `xlsx` and then `papaparse` processes that CSV string.

Reply:
@huddlej I can definitely add a comment! Whatever makes this clearer! 
Would it make more sense to have the comment here in `parseCsv.js` or in `metadata.js` where the function gets called?

Reply:
@joverlee521 A comment in `metadata.js` would probably be most useful. Once you get to `parseCsv.js`, as a code reader, you probably already know what's going on.

Reply:
I re-pushed with updated comments